conans/client/hook_manager.py

```diff
@@ -10,8 +10,13 @@
 from conans.util.files import save
 
 attribute_checker_hook = """
+from conans.model.conan_file import is_alias_conanfile
+
 def pre_export(output, conanfile, conanfile_path, reference, **kwargs):
     # Check basic meta-data
+    if is_alias_conanfile(conanfile):
```

Comment:
@danimtb I think built-in hooks are not updated automatically, are they? So existing users won't get this functionality.

Reply:
Yes, hooks with the same name are overwritten right now. So attribute_checker.py will be replaced with this new version